packages/server/src/queries/complex/pools/providers/sidecar.ts

```diff
@@ -1,43 +1,57 @@
-import { CoinPretty, PricePretty, RatePretty } from "@keplr-wallet/unit";
+import { CoinPretty, Dec, PricePretty, RatePretty } from "@keplr-wallet/unit";
 import { AssetList, Chain } from "@osmosis-labs/types";
 import { timeout } from "@osmosis-labs/utils";
 import cachified, { CacheEntry } from "cachified";
 import { LRUCache } from "lru-cache";
 
-import { IS_TESTNET } from "../../../../env";
+import { EXCLUDED_EXTERNAL_BOOSTS_POOL_IDS, IS_TESTNET } from "../../../../env";
 import { PoolRawResponse } from "../../../../queries/osmosis";
 import { queryPools } from "../../../../queries/sidecar";
 import { DEFAULT_LRU_OPTIONS } from "../../../../utils/cache";
 import { getAsset } from "../../assets";
 import { DEFAULT_VS_CURRENCY } from "../../assets/config";
 import { getCosmwasmPoolTypeFromCodeId } from "../env";
-import { Pool, PoolType } from "../index";
+import { Pool, PoolIncentiveType, PoolType } from "../index";
 
 type SidecarPool = Awaited<ReturnType<typeof queryPools>>[number];
 
 const poolsCache = new LRUCache<string, CacheEntry>(DEFAULT_LRU_OPTIONS);
 
+/**
+ * Pools that are excluded from showing external boost incentives APRs.
+ */
+const ExcludedExternalBoostPools: string[] =
+  (EXCLUDED_EXTERNAL_BOOSTS_POOL_IDS ?? []) as string[];
+
 /** Lightly cached pools from sidecar service. */
 export function getPoolsFromSidecar({
   assetLists,
   poolIds,
   minLiquidityUsd,
+  withMarketIncentives = true,
 }: {
   assetLists: AssetList[];
   chainList: Chain[];
   poolIds?: string[];
   minLiquidityUsd?: number;
+  withMarketIncentives?: boolean;
 }): Promise<Pool[]> {
   return cachified({
     cache: poolsCache,
     key:
       (poolIds ? `sidecar-pools-${poolIds.join(",")}` : "sidecar-pools") +
-      minLiquidityUsd,
-    ttl: 5_000, // 5 seconds
+      minLiquidityUsd +
+      withMarketIncentives.toString(),
+    ttl: 6_000, // 6 seconds
+    staleWhileRevalidate: 3_500, // 3.5 seconds
```

Comment:
Do you recall if this is safe to do in the edge environment? @jonator 

I remember us having issues with the flag in edge but unsure if it was because Imperator was taking overly long to load